crates/re_space_view_bar_chart/src/view_part_system.rs

```diff
@@ -20,6 +20,22 @@ impl ViewPartSystem for BarChartViewPartSystem {
         vec1::vec1![Tensor::name()]
     }
 
+    fn queries_any_components_of(
+        &self,
+        store: &re_arrow_store::DataStore,
+        ent_path: &EntityPath,
+        _components: &[ComponentName],
+    ) -> bool {
+        if let Some(tensor) = store.query_latest_component::<Tensor>(
```

Comment:
We should only do this if `Tensor` is in `_components`. The way we construct `components` at the moment guarantees that's going to be the case if `query_latest_component` is going to find something, but I can conceive of Blueprint+Auto hybrid scenarios where that might not be the case.